scripts/install.js

```diff
@@ -109,18 +109,68 @@ function checkAndDownloadBinary() {
       return;
     }
 
-    download(sass.getBinaryUrl(), sass.getBinaryPath(), function(err) {
-      if (err) {
-        console.error(err);
-        return;
-      }
+    var tmpPath = getTempPath(sass.getBinaryName());
+    if (!(sass.hasBinary(tmpPath))) { // download and install
+      download(sass.getBinaryUrl(), tmpPath, function(err) {
+        if (err) {
+          console.error(err);
+          return;
+        }
+        console.log('Binary downloaded at', tmpPath);
+        copyBinary(tmpPath, sass.getBinaryPath());
+      });
 
-      console.log('Binary downloaded and installed at', sass.getBinaryPath());
-    });
+    } else { // install only
+      copyBinary(tmpPath, sass.getBinaryPath());
+    }
   });
 }
 
 /**
+ * Find a temp folder for file
+ *
+ * @param {String} binaryName
+ * @returns {string}
+ * @api private
+ */
+
+function getTempPath(binaryName) {
+  var candidateTmpDirs = [
+    process.env.TMPDIR || process.env.TEMP || process.env.npm_config_tmp,
+    '/tmp',
+    path.join(process.cwd(), 'tmp')
+  ];
+
+  for (var i = 0; i < candidateTmpDirs.length; i++) {
+    var candidatePath = path.join(candidateTmpDirs[i], 'node-sass', 'releases', 'download', 'v' + pkg.version);
+
+    try {
+      mkdir.sync(candidatePath);
+      return path.join(candidatePath, binaryName);
+    } catch (err) {
+      console.error(candidatePath, 'is not writable:', err.message);
+    }
+  }
+}
+
+/**
+ * Copy file
+ *
+ * @param tmp
+ * @param dest
+ * @api private
+ */
+function copyBinary(tmp, dest) {
```

Comment:
This function is named specifically for copying the binaries so you can make the binary name it's responsibility.

```js
var from = path.join(tmp, sass.getBinaryName());
```